README.md

```diff
@@ -1 +1,78 @@
-# sumologic-kubernetes-collection
\ No newline at end of file
+# sumologic-kubernetes-collection
+
+# Installation
+
+First use `kops` to deploy your Kubernetes cluster in your AWS account.
```

Comment:
This looks more like a document for internal use. Remember customer don't rely on `kops` or `aws` to deploy their k8s cluster

Reply:
You're right. We should leave this PR unmerged until we can polish it for customers with the help of the Doc team and Frank.